xml/admin_ceph_gateway.xml

```diff
@@ -3279,365 +3211,211 @@ rgw_zone=us-east-1</screen>
-          {
-              "key": "83859a9a-9901-4f00-aa6d-285c777e10f0",
-              "val": 630926044
-          },
-          {
-              "key": "950c1a43-6836-41a2-a161-64777e07e8b8",
-              "val": 4276257543
-          },
-          {
-              "key": "d9522067-cb7b-4129-8751-591e45815b16",
-              "val": 329470157
-          }
-      ]
-  },
-  "master_zonegroup": "d4018b8d-8c0d-4072-8919-608726fa369e",
-  "master_zone": "83859a9a-9901-4f00-aa6d-285c777e10f0",
-  "period_config": {
-      "bucket_quota": {
-          "enabled": false,
-          "max_size_kb": -1,
-          "max_objects": -1
-      },
-      "user_quota": {
-          "enabled": false,
-          "max_size_kb": -1,
-          "max_objects": -1
-      }
-  },
-  "realm_id": "4a367026-bd8f-40ee-b486-8212482ddcd7",
-  "realm_name": "gold",
-  "realm_epoch": 2
-}</screen>
-    <para>
-     Note that the period epoch number has incremented, indicating a change in
-     the configuration.
-    </para>
-   </sect3>
-   <sect3 xml:id="ceph-rgw-fed-seccluster-rgwstart">
-    <title>Start the &ogw;</title>
-    <para>
-     This is similar to starting the &ogw; in the first zone. The only
-     difference is that the &ogw; zone configuration should reflect the
-     <literal>us-west</literal> zone name:
-    </para>
-<screen>[client.rgw.us-west]
-rgw_frontends="beast port=80"
-rgw_zone=us-west</screen>
-    <para>
-     Start the second &ogw;:
-    </para>
-<screen>&prompt.root;systemctl start ceph-radosgw@rgw.us-west</screen>
+   <sect3 xml:id="ceph-rgw-check-sync-status">
+    <title>Check Synchronization Status</title>
+    <para>Once the secondary zone is up and running, check the synchronization
+      status. Synchronization copies users and buckets created in the
+      master zone to the secondary zone.</para>
+<screen>&prompt.cephuser;radosgw-admin sync status</screen>
+    <para>The output provides the status of synchronization operations.
+      For example:</para>
+<screen>realm f3239bc5-e1a8-4206-a81d-e1576480804d (gold)
+    zonegroup c50dbb7e-d9ce-47cc-a8bb-97d9b399d388 (us)
+         zone 4c453b70-4a16-4ce8-8185-1893b05d346e (us-west)
+metadata sync syncing
+              full sync: 0/64 shards
+              metadata is caught up with master
+              incremental sync: 64/64 shards
+    data sync source: 1ee9da3e-114d-4ae3-a8a4-056e8a17f532 (us-east)
+                      syncing
+                      full sync: 0/128 shards
+                      incremental sync: 128/128 shards
+                      data is caught up with source</screen>
+    <note>
+      <para>Secondary zones accept bucket operations; however, secondary zones
+        redirect bucket operations to the master zone and then synchronize with
+        the master zone to receive the result of the bucket operations.
+        If the master zone is down, bucket operations executed on the
+        secondary zone will fail, but object operations should succeed.</para>
```

Comment:
We have mentioned this before. And I'm ok with repeating but just pointing it out.

Should we give some bucket or metadata operations that wont work if the master zone is down? Maybe link to a bucket operations section. Or ignore this if it's muddies the scope of this doc too much.

Reply:
We don't specifically have a bucket operations section. So I'm not sure what you're suggesting to link to? https://docs.ceph.com/docs/master/radosgw/multisite/

Reply:
@theanalyst any thoughts?

Reply:
I don't mind keeping it here again, we do mention this in part at the beginning about user creation and metadata, but here it seems fine in view of other info about objects etc.